share/tools/web_config/fishconfig.css

```diff
@@ -91,25 +94,25 @@ body {
     height: 30px;
 }
 
-.master_detail_table {
-    display: table;
-    margin-top: 10px;
+.function_tab {
     margin-left: 12px;
     margin-right: 12px;
+    display: flex;
+    flex-direction: row;
+    height: calc(100vh - 129px);
```

Comment:
Can we do without the hard-coded values here? I think we can find a workaround that doesn't involve subtracting the height; usually by adding another layer of div wrappers for everything.

Reply:
After looking at the css, I see this is the only child of an element with min-height:90hv, so in my new push, Iset height to 90hv